Example/PaymentSheet Example/PaymentSheet Example/PaymentSheetTestPlayground.swift

```diff
@@ -113,7 +114,28 @@ struct PaymentSheetTestPlayground: View {
         } set: { newString in
             playgroundController.settings.customCtaLabel = (newString != "") ? newString : nil
         }
-
+    }
+    var customerModeBinding: Binding<PaymentSheetTestPlaygroundSettings.CustomerMode> {
+        Binding<PaymentSheetTestPlaygroundSettings.CustomerMode> {
+            return playgroundController.settings.customerMode
+        } set: { newMode in
+            playgroundController.settings.customerId = nil
+            playgroundController.settings.customerMode = newMode
+        }
+    }
+    var customerIdBinding: Binding<String> {
+        Binding<String> {
+            switch playgroundController.settings.customerMode {
+            case .guest:
+                return ""
+            case .new:
+                return playgroundController.settings.customerId ?? ""
+            case .returning:
+                return playgroundController.settings.customerId ?? ""
+            }
+        } set: { newString in
+            playgroundController.settings.customerId = (newString != "") ? newString : nil
```

Comment:
When we set the customerId manually, should we update the customerMode to something, like new? What if I enter a customerId and I'm in guest mode.

Reply:
There are some issues with manually typing in the customer id.  For now, I've disabled it until I can get back to this.